_topic_maps/_topic_map.yml

```diff
@@ -392,6 +392,8 @@ Topics:
     File: ipi-install-prerequisites
   - Name: Setting up the environment for an OpenShift installation
     File: ipi-install-installation-workflow
+  - Name: Installing OpenShift
```

Comment:
I recommend that you use "Installing a cluster" to align with the other installation methods.

Reply:
Thanks. Fixed.